### PR TITLE
Update links for the domain change

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Update links for the OS website domain change

**Related issue(s)**

### CODE REVIEW

File: installation/resources/crds/telemetry/logpipelines.crd.yaml [hunk 1]
```diff
@@ -341,7 +341,7 @@ spec:
                 type: array
               unsupportedMode:
                 description: Is active when the LogPipeline uses a `custom` output
-                  or filter; see [unsupported mode](https://kyma-project.io/docs/kyma/main/01-overview/telemetry/telemetry-02-logs#unsupported-mode).
+                  or filter; see [unsupported mode](https://kyma-project.io/#/telemetry-manager/user/02-logs?id=unsupported-mode).
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply installation/resources/crds/telemetry/logpipelines.crd.yaml]: Ok, I replaced the colon with a hyphen :)

**Review by IwonaLanger — DISMISSED** [review]

**Review by mrCherry97 — DISMISSED** [review]

File: resources/serverless/README.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ This project contains the chart for the Function Controller.
 
 ## Details
 
-To learn how Serverless is used in Kyma, see its [official documentation](https://kyma-project.io/docs/kyma/latest/01-overview/serverless/).
+To learn how Serverless is used in Kyma, see its [official documentation](https://kyma-project.io/#/01-overview/serverless/README).
```
Comment (k15r) [inline resources/serverless/README.md:15]: i see a new `README` here. is this on purpose?

Comment (mmitoraj) [reply resources/serverless/README.md:15]: It is the same content aggregated from 7 files in one pre-existing README to simplify navigation.

File: resources/eventing/charts/controller/templates/busola-extension.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@ data:
     category: Configuration
     scope: namespace
     description: >-
-      {{"{{[Subscription](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/evnt-01-subscription#documentation-content)}}"}} is used to subscribe to events.
+      {{"{{[Subscription](https://kyma-project.io/#/05-technical-reference/00-custom-resources/evnt-01-subscription#documentation-content)}}"}} is used to subscribe to events.
```
Comment (muralov) [inline resources/eventing/charts/controller/templates/busola-extension.yaml:20]: this one is navigating to the main page too. Not to the specific page.

File: README.md [hunk 4]
```diff
@@ -13,11 +13,11 @@
...
-- [Eventing](https://kyma-project.io/docs/kyma/latest/01-overview/eventing/) that provides messaging channel to receive events, enrich them, and trigger business flows using Functions or services
-- [Service Mesh](https://kyma-project.io/docs/kyma/latest/01-overview/service-mesh/) for service-to-service communication and proxying
-- [Service Management](https://kyma-project.io/docs/kyma/latest/01-overview/service-management/smgt-01-overview/) to use the built-in cloud services from such cloud providers as GCP, Azure, and AWS
+- [Serverless](https://kyma-project.io/#/01-overview/serverless/README) development platform to run lightweight Functions in a cost-efficient and scalable way
+- [System connectivity](https://kyma-project.io/#/01-overview/application-connectivity/README) that provides endpoint to securely register Events and APIs of external applications
+- [Eventing](https://kyma-project.io/#/01-overview/eventing/README) that provides messaging channel to receive events, enrich them, and trigger business flows using Functions or services
```
[the remark this answers is not in the document]
Comment (mmitoraj) [reply README.md:18]: The link will take you to the Eventing section -> https://kyma-project.github.io/kyma/#/01-overview/eventing/README. As I said, the domain has not been changed yet so the links you see here don't work.

**Review by muralov — APPROVED** [review]

**Review by dbadura — DISMISSED** [review]

**Review by dbadura — APPROVED** [review]

**Review by muralov — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by mrCherry97 — DISMISSED** [review]

File: docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md [hunk 5]
```diff
@@ -148,6 +148,6 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | **conditions.&#x200b;lastTransitionTime**  | string | An array of conditions describing the status of the pipeline. |
 | **conditions.&#x200b;reason**  | string | An array of conditions describing the status of the pipeline. |
 | **conditions.&#x200b;type**  | string | The possible transition types are:<br>- `Running`: The instance is ready and usable.<br>- `Pending`: The pipeline is being activated. |
-| **unsupportedMode**  | boolean | Is active when the LogPipeline uses a `custom` output or filter; see [unsupported mode](https://kyma-project.io/docs/kyma/main/01-overview/telemetry/telemetry-02-logs#unsupported-mode). |
+| **unsupportedMode**  | boolean | Is active when the LogPipeline uses a `custom` output or filter; see [unsupported mode](../../01-overview/telemetry/telemetry-02-logs.md#unsupported-mode). |
```
Comment (chrkl) [inline docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md:151]: Having the absolute URL here was a conscious decision. This table is generated from our CRD description fields and the relative path would be shown for instance in Busola or when using `kubectl explain`. It's not optimal, but at the moment we do not see a better solution.

Comment (mmitoraj) [reply docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md]: I undid the change

**Review by chrkl — DISMISSED** [review]

**Review by dbadura — APPROVED** [review]

**Review by IwonaLanger — DISMISSED** [review]

**Review by mrCherry97 — DISMISSED** [review]

**Review by muralov — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]

**Review by chrkl — APPROVED** [review]

**Review by muralov — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**